src/Umbraco.Core/Trees/ITree.cs

```diff
@@ -3,7 +3,7 @@
     // TODO: we don't really use this, it is nice to have the treecontroller, attribute and ApplicationTree streamlined to implement this but it's not used
     // leave as internal for now, maybe we'll use in the future, means we could pass around ITree
     // TODO: We should make this a thing, a tree should just be an interface *not* a controller
-    internal interface ITree
+    public interface ITree
```

Comment:
Have made this public such that its accessible to `Umbraco.Web.BackOffice`.  We could use `InternalsVisibleTo` here if it's preferred not to make this public.

Reply:
When used by our own non-test dlls, we prefere it to be public :)